PyPortal_Azure_Plant_Monitor/azure_gfx_helper.py

```diff
@@ -66,19 +66,19 @@ def __init__(self, is_celsius=True):
         self.moisture_label.y = 135
         self._text_group.append(self.moisture_label)
 
-        self.moisture_text = Label(self.data_font, max_glyphs=10)
+        self.moisture_text = Label(self.data_font, max_glyphs=10, text="")
         self.moisture_text.x = 200
         self.moisture_text.y = 175
         self._text_group.append(self.moisture_text)
 
-        self.azure_status_text = Label(self.main_font, max_glyphs=15)
+        self.azure_status_text = Label(self.main_font, max_glyphs=15, text="")
```

Comment:
This is a workaround for https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/issues/45. This can be reverted once the fix is released.